client/src/components/record/video-player.tsx

```diff
@@ -71,19 +72,23 @@ function VideoPlayer(props: {
       return;
     }
     const duration = sliderToVideoDuration();
-    if (duration && state.played >= trim[1] / 100) {
+    if (duration && state.played >= trim[1] / 100 && !trimInProgress) {
       reactPlayerRef.current.seekTo(duration[0]);
     }
   }
 
   function onUpdateTrim(value: number | number[]): void {
     if (!trimInProgress) setTrimInProgress(true);
-    if (!Array.isArray(value) || !reactPlayerRef?.current) {
+    if (
```

Comment:
is confusing what `value` is here since it can be essentially a scalar number or what I guess is really a tuple of two numbers (start, end)? What is the case for it ever being a scalar?

Reply:
Clarified variable and data type in commit: 0e3e873
It would be a scalar if the Slider component was a single value slider with one knob instead of a range slider with 2 knobs.